doc/manual/meson.build

```diff
@@ -0,0 +1,75 @@
+ignore_headers = [
+  'private.h',
+  'pkcs11.h',
+  'conf.h',
+  'debug.h',
+  'dict.h',
+  'filter.h',
+  'log.h',
+  'mock.h',
+  'modules.h',
+  'pkcs11.h',
+  'pkcs11i.h',
+  'pkcs11x.h',
+  'private.h',
+  'client.h',
+  'proxy.h',
+  'rpc.h',
+  'rpc-message.h',
+  'util.h',
+  'virtual.h',
+  'virtual-fixed.h',
+  'array.h',
+  'compat.h'
+]
+
+content_files = [
+  'p11-kit-config.xml',
+  'p11-kit-sharing.xml',
+  'p11-kit-devel.xml',
+  'p11-kit-proxy.xml',
+  'p11-kit-trust.xml',
+  'p11-kit-remoting.xml',
+  'p11-kit.xml',
+  'pkcs11.conf.xml',
+  'trust.xml',
+  'annotation-glossary.xml'
+]
+
+configure_file(
```

Comment:
Both [sysdir.xml.in](https://github.com/p11-glue/p11-kit/pull/231/files?file-filters%5B%5D=.am&file-filters%5B%5D=.build&file-filters%5B%5D=.c&file-filters%5B%5D=.in&file-filters%5B%5D=.map&file-filters%5B%5D=.sh&file-filters%5B%5D=.txt&file-filters%5B%5D=No+extension&owned-by%5B%5D=#diff-6d687cd02f22ad68f0cfcee32026fae5) and [userdir.xml.in](https://github.com/p11-glue/p11-kit/pull/231/files?file-filters%5B%5D=.am&file-filters%5B%5D=.build&file-filters%5B%5D=.c&file-filters%5B%5D=.in&file-filters%5B%5D=.map&file-filters%5B%5D=.sh&file-filters%5B%5D=.txt&file-filters%5B%5D=No+extension&owned-by%5B%5D=#diff-30761a444d2282b18855a00b7f326697) are used to store just a path. Why not use just one file and replace it with the proper path?

This can also be done for `version.xml` file.

```
echo '@VALUE@' > content.xml.in
```
```suggestion
in_files = [
  {'name': 'sysdir.xml', 'value': p11_system_config},
  {'name': 'userdir.xml', 'value': p11_user_config},
  {'name': 'version.xml', 'value': meson.project_version()},
]

foreach in_file: in_files
  content_files += configure_file(
    input: 'content.xml.in',
    output: in_file['name'],
    configuration: {'VALUE': in_file['value']})
endforeach
```

This way you will reduce 3 template files to just one. (BTW: Although not necessary because they are already build in the proper directory but these generated files are also content files, that's why the `content_files +=`).

Reply:
Thank you, but for me it looks a bit over-engineered.